### DISCUSSION TITLE
RBS files for models?

### DISCUSSION THREAD

**Discussion by rgalanakis** [jeremyevans/sequel#1869] [General]
I'm wondering if anyone has done an RBS generator for Sequel model types? We already use `sequel-annotate`, which has a somewhat different purpose. But I have recently found myself using YARD annotations more, and it seems like we should be able to generate RBS from the Sequel model class metadata. Has anyone tried this? Is anyone working on it? Is it going to be much more difficult than I expect?

**Comment by adam12** [comment]
I've been interested in this but haven't had an opportunity to dig into it. I had presumed I might try to pair it with the `schema_caching` plugin if it contained the necessary information.
